ui/component/wunderbarSuggestions/view.jsx

```diff
@@ -228,12 +229,19 @@ export default function WunderBarSuggestions(props: Props) {
     function handleKeyDown(event) {
       const { ctrlKey, metaKey, keyCode } = event;
 
+      if (keyCode === K_KEY_CODE && ctrlKey) {
+        inputRef.current.focus();
+        inputRef.current.select();
+        return;
+      }
+      
       if (!inputRef.current) {
         return;
       }
 
       if (inputRef.current === document.activeElement && keyCode === ESC_KEY_CODE) {
-        inputRef.current.blur();
+        inputRef.current.value = "";
+        inputRef.current.focus();
```

Comment:
Previously, pressing escape would blur the search focus but leave the text.
Now we're clearing the text, but keeping the focus. 

Youtube seems to clear the suggestion box on first ESC, and then blur focus on second ESC press. 
Does this seem like a good idea?

Reply:
I implemented a hybrid of the two where escape will clear the text and keep the wunderbar in focus if there was any text in the box.  If there is no text in the box then the box will lose focus.  I think this is a good compromise.  It keeps the old behavior if you press escape twice and also keeps the clearing of text for faster searching.